apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2Test.java

```diff
@@ -576,8 +576,10 @@ private static void verifyCodedExtension(String url, String code, String display
     assertTrue(compare.equalsDeep(ex));
   }
 
-  private static void verifyCoverageStatus() {
-    assertEquals("active", coverage.getStatus().toCode());
+  private static void verifyCoverageStatus(String status) {
+    if (status != null && status.length() > 0) {
```

Comment:
do we actually need this condition? I would think that if a test passed in null or an empty string here then this method should assert that the coverage code is such. If that is wrong then the test should not assert it as so.

The risk I see here is that if a test does pass null, this will not assert anything and silently just work.

Reply:
yup with the refactor of the method this was no longer needed